addon/templates/components/validated-input.hbs

```diff
@@ -82,7 +83,9 @@
           checked    = {{get model name}}
           onclick    = {{action "update" value = "target.checked"}}
           onfocusout = {{action "setDirty"}}>
-        <span>{{label}}</span>
+        <span style="display: inline-block;">
+          {{component (if labelComponent labelComponent "validated-label") label=label config=config inputId=inputId}}
+        </span>
```

Comment:
This would render a `<label>` inside another `<label>` - which is not exactly what we want here, right?
Maybe it would make sense to use this style of checkbox labels instead:
```html
<input type="checkbox" id="coding" name="interest" value="coding">
<label for="coding">Coding</label>
```
(taken from [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox))

With this style, it seems that we could use `labelComponent` / `validated-label` exactly in the same way as with the other inputs. And it looks like all that needs to be changed is that the enclosing `<label>` tag has to be removed.

What do you think? :smile:

Reply:
Edit: Just realized that this is what `isLabel` is supposed to do. What do you think about using the `label for="..."` syntax instead, so that we don't need `isLabel`?